### PR TITLE
Added white logo

### PR DESCRIPTION
**Pull request by leelatanniru**
This fixes the issue #1073

### CODE REVIEW

**Comment by jamessspanggg** [comment]
Note: Merge this after https://github.com/thenewboston-developers/Marketing/pull/12 is merged

File: src/containers/Assets/index.tsx [hunk 1]
```diff
@@ -20,12 +21,22 @@ const assets: Asset[] = [
     imageUrl: TnbLogoAndWordmark,
     subtext: 'TNB-Logo-and-Wordmark.Zip',
   },
+  {
+    downloadLink: `${socialMediaUrls.github}/Marketing/raw/master/Logo-and-Wordmark-white.zip`,
+    imageUrl: TnbLogoAndWordmarkWhite,
+    subtext: 'TNB-Logo-and-Wordmark-white.Zip',
```
Comment (jamessspanggg) [inline src/containers/Assets/index.tsx:27]: Just a suggestion: should we add a new property `color: light` or `color: dark` to each asset? this is to account for future assets that may also have light colors. Then the classname of the div should be dynamic like:

```
className={clsx('Assets__card-wrapper', {
          'Assets__card-wrapper--white': asset.color === 'light',
        })}
```

Not too sure about this tho. Let's see what the leads think about this.

**Comment by manishram** [comment]
> Note: Merge this after [thenewboston-developers/Marketing#12](https://github.com/thenewboston-developers/Marketing/pull/12) is merged

Merged

**Review by jamessspanggg — APPROVED** [review]
LGTM!

thanks for the tidy up @buckyroberts :) enjoying the stream

**Pull request merged**